sam/cigar.go

```diff
@@ -197,6 +196,21 @@ func init() {
 	}
 }
 
+var powers = []int{1, 10, 100, 1000, 10000, 100000, 1000000, 10000000, 100000000}
+
+// atoi is an internal function; bytes in b are known to be digits.
```

Comment:
`// atoi returns the integer interpretation of b which must be an ASCII decimal number representation.`